packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-profit-loss-text.tsx

```diff
@@ -16,24 +27,29 @@ const AccumulatorsProfitLossText = ({
     currency,
     className = 'sc-accumulators-profit-loss-text',
     profit,
-}) => {
+}: TAccumulatorsProfitLossText) => {
     const [is_fading_in, setIsFadingIn] = React.useState(false);
     const [is_sliding, setIsSliding] = React.useState(false);
-    const prev_profit = React.useRef(profit);
+    const prev_profit = React.useRef<number>(profit);
     const prev_profit_tenth = +prev_profit.current?.toFixed(2).split('.')[1][0];
     const [current_profit_tenth, setCurrentProfitTenth] = React.useState(prev_profit_tenth);
-    const profit_tenth_ref = React.useRef();
-    const interval_id_ref = React.useRef(null);
-    const fading_in_timeout_id = React.useRef();
-    const sliding_timeout_id = React.useRef();
+    const profit_tenth_ref = React.useRef<number>(0);
```

Comment:
```suggestion
    const profit_tenth_ref = React.useRef(0);
```
We can omit type declaration here because TS can infer (figure out) types successfully based on initial primitive values we pass to useState() 😊